README.md

```diff
@@ -5,6 +5,8 @@ Provides modules for [Ansible](https://www.ansible.com/community) to manage [Mik
 
 You can find [documentation for the modules and plugins in this collection here](https://ansible.fontein.de/collections/community/routeros/).
 
+Please note that this collection does **not** support Windows targets for the modules not using the `network_cli` connection.
```

Comment:
Sorry, I can't seem to grasp the meaning. **Windows target** here is a host machine on which Ansible is being run, right?

Reply:
Target is the machine where the module is run on, when using the regular connection plugin. When using the network_cli connection, the modules are run locally (and connect to a network device), so for them there's technically no need to document this. But the `api` module is a regular module and - when not `delegate_to: localhost` is added or `localhost` is used - will run on whatever target the user is using (which cannot be a RouterOS device, but for example a jump host in a network which has access to the RouterOS device). Basically the above says "we do not support Windows jump hosts for the `community.routeros.api` module".

I guess we need to formulate this more clearly though :)

Reply:
Got it, thanks! Should we maybe just straight-up say it then, I think this is perfect:

> ### Known issues / edge cases
> Please note that `community.routeros.api` module does **not** support Windows jump hosts!

Reply:
It's great! 👍🏻